registration/static/js/src/registration.js

```diff
@@ -49,20 +64,22 @@ app.controller('Registration', ['$scope', '$http', 'djangoForm', function($scope
         return _.keys($scope.form).filter(key => !/^\$/.test(key));
```

Comment:
@omarkhan The `=>` function notation does not have very good browser support yet. It doesn't work with latest Safari on OS X, for example.